docs/cli.md

```diff
@@ -102,7 +102,7 @@ npx babel src --out-file script-compiled.js
 Ignore spec and test files
 
 ```sh
-npx babel src --out-dir lib --ignore spec.js,test.js
+npx babel src --out-dir lib --ignore "src/**/*.spec.js","src/**/*.test.js"
```

Comment:
`"src/**/*.spec.js,src/**/*.test.js"` works too but that's about the same.

Reply:
Good to know. Thanks!